src/components/MenuItem.tsx

```diff
@@ -141,6 +141,11 @@ type MenuItemBaseProps = {
     /** A description text to show under the title */
     description?: string;
 
+    /** Text to show below menu item. This text is not interactive */
+    helperText?: string;
+
+    helperTextStyle?: StyleProp<TextStyle>;
```

Comment:
Should we also add a comment for this one?

Reply:
nice catch